packages/platform-browser/src/browser/browser_adapter.ts

```diff
@@ -57,6 +57,7 @@ export class BrowserDomAdapter extends GenericBrowserDomAdapter {
     return node instanceof DocumentFragment;
   }
 
+  /** @deprecated No longer being used. To be removed in a future version. */
```

Comment:
nit: since VE code still uses that, we can just mention that it's unused in Ivy here (and in a couple other places updated in this PR):
```suggestion
  /** @deprecated No longer being used in Ivy code. To be removed in a future version. */
```